src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/modes/Recast.java

```diff
@@ -33,27 +32,33 @@ public Recast() {
     public static boolean rubberbanded = false;
 
     int tickDelay = elytraFly.restartDelay.get();
+    double prevFov = mc.options.getFovEffectScale().getValue();
```

Comment:
? why

Reply:
Because it sprints every time it touches the ground (every 1 second), and if the player has fov effects enabled, the camera is just gonna zoom in and out.

Reply:
or you can just send sprint packets to prevent the fov change happen

Reply:
Tried it, doesn't work.